pom.xml

```diff
@@ -659,5 +662,13 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>plugins</id>
```

Comment:
`plugins` is a little bit generic to me. Rename it to `encoding-plugin` or `vector-plugins`? Any suggestion?  @gszadovszky 

In addition, please rename `plugins` folder to `parquet-plugins` to follow naming of other sub-directories.

Reply:
@wgtmac  @gszadovszky  vector-plugins +1, and it can show the feature of this PR. 
 I have renamed plugins to parquet-plugins.